pontoon/localizations/templates/localizations/localization.html

```diff
@@ -18,6 +18,9 @@
 {% block heading %}
 <section id="heading">
   <div class="container clearfix">
+      {{ HeadingInfo.download_button(
+        locale.code, project.slug)
+      }}
```

Comment:
Nit: You can put these three lines in just one.

Reply:
Okay,Suppose if i was having a link in this macro then we would prefer using the change of line instead of writing everything in a single line?